### DISCUSSION TITLE
Looks like renaming a column does not update the internal schema

### DISCUSSION THREAD

**Discussion by Pachat** [Budibase/budibase#2940] [Help]
Hi,
Thanks for the September update. Very much appreciated.

While testing on a new app, I made a typo on a column name.
After correction, the right name appeared on top of the column of the table, but the old name was kept in the forms.
As a consequence, the data already there did not appear in the cell.  
And did not save in New or Edit form.

So the situation is that I see the right data in app_dev but not in app. Even after publishing.
Being not reflected in app, app_dev does not permit to see what is going to be in app.

To solve this, it is necessary to go on every modified field in every form to update the drop-down Field.
It would be much easier to have this done while saving the new name (as it is for the column name on top of the table).

Or did i miss something?

**Comment by mjashanks** [comment]
Hey @Pachat 

Yes, you will have to go to every field that uses the form and update it. It would be nice to update them all, but it's actually quite a complex task to do. We would have to look through every single binding in the app to do it properly. It's also quite an infrequent action - so we are probably not going to be solving this anytime soon.

It does seem a bit strange that the issue does not happen after publishing though!

Thanks for reporting this though :)

**Comment by Pachat** [comment]
Thanks for answering. May be I was not clear: the issue is still there after publishing.

**Reply by mjashanks** [reply]
oh, so you fixed it on all the fields & bindings, then published - and the issue still persists in the published app?
